### PR TITLE
Added the --close-to-tray command line option.

### PR DESCRIPTION
**Pull request by LuckyThirteen**
If the ~~` --keep-alive`~~ `--close-to-tray` command line option is set, clicking on the `close` button won't kill the application, it will only hide the window.

### CODE REVIEW

**Comment by twolfson** [comment]
Any reason why we renamed this from `--close-to-tray` to `--keep-alive`? I feel like `keep-alive` is more confusing than helpful =/

File: app/js/index.js [hunk 1]
```diff
@@ -39,18 +52,48 @@
...
+	};
+
+  gui.App.on('open', function (cmdline) {
+		showWindow();
+  });
+
+  if (program.keepAlive) {
+		win.on('close', function () {
+			hideWindow();
+		});
```
Comment (twolfson) [inline app/js/index.js]: We can compact this to:

```js
win.on('close', hideWindow);
```

**Comment by LuckyThirteen** [comment]
I renamed it because when I think of "close to tray" I don't expect the the window to be visible in the taskbar... but that's only true if the other option is set too.

I can rename ofc, just say the word :)
(I'll also shorten the code like you suggested.)

**Comment by twolfson** [comment]
Well, I was imagining that the `--close-to-tray` takes both the `--minimize-to-tray` behavior and combines it with this one (e.g. if statements become `if (program.minimizeToTray || program.closeToTray)`). From a usage perspective, I don't think anyone who uses minimize/restore would expect their close to perform a minimize.

**Comment by LuckyThirteen** [comment]
Ah, I see! 
Well, I though the keep-alive could make sense for someone who prefers to see the window in the task bar also, it's just making sure you don't kill the app even if you click on the close (mostly by habit I guess, at least that's what I'm doing most of the time), normally for a chat like slack you don't want that.

Anyways, I'll combine them & rename the option, sorry for being slow at understanding things :)

**Comment by twolfson** [comment]
Once the requested changes in #68 are addressed and echoed in this PR, then I think everything will be fine. Please address #68 first though =X

**Comment by twolfson** [comment]
Since #112 has been landed, we no longer need this PR.

**Pull request closed without merging**